### PR TITLE
fix: add support of placing properties at root in af

### PR DESCRIPTION
**Pull request by aashishmalik**
## What are the changes introduced in this PR?

send properties under eventValue at root level, driving this via a config to maintain backward compatibility.

## What is the related Linear task?

Resolves INT-1431

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3082/test-report.html

File: src/v0/destinations/af/transform.js [hunk 1]
```diff
@@ -196,7 +203,7 @@ function processSingleMessage(message, destination) {
   let payload;
   switch (messageType) {
     case EventType.TRACK: {
-      payload = processEventTypeTrack(message);
+      payload = processEventTypeTrack(message, destination.Config.placePropertiesAtRoot);
```
Comment (ItsSudip) [inline src/v0/destinations/af/transform.js:206]:
```suggestion
      payload = processEventTypeTrack(message, destination.Config?.placePropertiesAtRoot);
```

Comment (saikumarrs) [inline src/v0/destinations/af/transform.js:206]: More appropriate naming.
```suggestion
      payload = processEventTypeTrack(message, destination.Config.addPropertiesAtRoot);
```

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3082/test-report.html

**Review by Gauravudia — APPROVED** [review]

**Review by Gauravudia — APPROVED** [review]

**Pull request merged**